### ISSUE TITLE
A11y - No way to set a `alt` tag on QImg

### ISSUE THREAD

**Issue by Evertvdw** [quasarframework/quasar#16227]
### What happened?

The alt property of QImg will get rendered as an `aria-label` and there is no way currently to use an alt tag on an image. My understanding is that you will need to supply an empty `alt=""` to signal that an image is not important. That cannot be done currently.

### What did you expect to happen?

The alt property should render an actual alt attribute, and a separate aria-label property for the people that want to specifically use an aria-label.

### Reproduction URL

https://codepen.io/Mett-codepens/pen/MWZYEbX?editors=101

### How to reproduce?

Inspect the example and see there is an `aria-label="hello"` attribute on the image.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar), Accessibility [a11y] (quasar)

**Comment by rstoenescu** [comment]
Hi,

The image element itself is set with aria-hidden="true" so it doesn't matter if it has "alt" or not. It should be ignored regardless.

Please reopen or comment if I am missing something.
